### DISCUSSION TITLE
napoleon?

### DISCUSSION THREAD

**Discussion by andypexai** [sphinx-extensions2/sphinx-autodoc2#16] [Ideas]
Is this compatible with Numpy/Google-style docstrings from sphinx.ext.napoleon?  I tried and it didn't seem like it.  I'm sort of locked into that style more than Myst docstrings at the moment.

**Comment by chrisjsewell** [comment]
Heya not at the moment I'm afraid, but it should be possible, see bottom of https://github.com/executablebooks/MyST-Parser/issues/228

**Reply by andypexai** [reply]
That's very informative, thank you.  I've got too much invested in napoleon to move away from that but I like your project and I'm thankful for your efforts.  Despite not doing exactly what I'd hoped, the autodoc2 cli was magnificent, and I had rendered html on the first try.

**Comment by cmarqu** [comment]
See also #33.
